db/seeders/correspondence_type_seeder.rb

```diff
@@ -34,5 +35,14 @@ def seed!
                 external_time_limit: 30,
                 deadline_calculator_class: 'CalendarDays')
 
+    rec = CorrespondenceType.find_by(abbreviation: 'OFFENDER')
+    rec = CorrespondenceType.new if rec.nil?
+    rec.update!(name: 'Offender SAR (OFFENDER)',
+                abbreviation: 'OFFENDER',
+                escalation_time_limit: 3,
+                internal_time_limit: 10,
+                external_time_limit: 20,
+                deadline_calculator_class: 'CalendarDays')
   end
```

Comment:
What is `rec`?

Is this duplicating the current Data Migration? (the `abbreviation` should be `OFFENDER_SAR`)

Reply:
Yeah, I don't know what "rec" stands for - "record" perhaps? It's the same as all the other ones in this file, so just keeping it consistent

The abbreviation shouldn't be OFFENDER_SAR until your branch is merged :)